connectors/qualys/configurations/config.json

```diff
@@ -0,0 +1,31 @@
+{
+    "connection": {
+        "schedule": {

```

Comment:
We dont need schedule field, if we are fine with default option which is the case here

Reply:
Schedule field is required. The Qualys Agent will send collected information every 4 hours. With default options ( 5m,10m,...) we will get less updates from the agent, so we are configuring schedulers with minimum interval of 1h,2h,4h,6h,... to get reasonable data to import.